src/redux/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  greeting: '',
+  error: null,
+};
+
+export const getRandomGreeting = createAsyncThunk('getRandomGreeting', async () => {
+  try {
+    const response = await axios.get('http://127.0.0.1:3000/api/v1/greetings');
+    const { data } = response;
+    return data;
+  } catch (error) {
+    return error.message;
+  }
+});
+
+export const greetingsSlice = createSlice({
+  name: 'greetings',
+  initialState,
+  reducers: {
+  },
+  extraReducers: (builder) => {
+    builder
+      .addCase(getRandomGreeting.fulfilled, (state, action) => {
```

Comment:
- [ ] Nice work on adding reducer on fetching the data from the API, but I think it would me also good to ad the cases where the request failed, and pending, sothat the user knows what happend on each requests.